src/kubeobjects/labels.go

```diff
@@ -110,9 +110,9 @@ func (labels *AppLabels) BuildMatchLabels() map[string]string {
 	}
 }
 
-func MergeLabels(labels ...map[string]string) map[string]string {
+func MergeMap(maps ...map[string]string) map[string]string {
```

Comment:
there is a `map.go` file, this function would fit it perfectly 🥲

Reply:
:smiling_face_with_tear: